application/account-management/WebApp/src/main.tsx

```diff
@@ -1,17 +1,20 @@
+import { i18n } from "@lingui/core";
+import { I18nProvider } from "@lingui/react";
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { I18nProvider } from "@lingui/react";
-import { i18n } from "@lingui/core";
+import { ApplicationInsightsProvider } from "./lib/applicationInsights/ApplicationInsightsProvider";
+import { ApplicationInsightsReactFilesystemRouter } from "./lib/applicationInsights/ApplicationInsightsReactFilesystemRouter";
 import "./main.css";
-import { ReactFilesystemRouter } from "@platformplatform/client-filesystem-router/react";
 import { dynamicActivate, getInitialLocale } from "./translations/i18n";
 
 await dynamicActivate(i18n, getInitialLocale());
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
     <I18nProvider i18n={i18n}>
-      <ReactFilesystemRouter />
+      <ApplicationInsightsProvider>
+        <ApplicationInsightsReactFilesystemRouter />
```

Comment:
Somehow this new `<ApplicationInsightsReactFilesystemRouter />` changes the UI, so the navigation bar is not taking up the full hight of the screen. If I revert to just `<ReactFilesystemRouter />` it looks good again.

Reply:
Removed React Component tracking for now